gcloud/streaming/test_transfer.py

```diff
@@ -204,7 +204,7 @@ def test_FromFile_w_existing_file_w_override_wo_auto_transfer(self):
             self.assertFalse(download.auto_transfer)
             del download  # closes stream
             with open(filename, 'rb') as fileobj:
-                self.assertEqual(fileobj.read(), '')
```

Comment:
That change came in via 1378947, "Fix Py3k testing glitches".  Until the renaming, the tests weren't being found by Travis-tox, and therefore some Py3k issues remained.